content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/fedex.rst

```diff
@@ -0,0 +1,165 @@
+which can be used to configure a shipping method for testing purposes.
+
+Click on :guilabel:`Get Production Key` to start the process of getting a developer key and meter
+number which can be used to configure a shipping method that generates real shipments with FedEx.
+
+After clicking either option, follow the instructions until the :guilabel:`Confirmation` screen is
+reached. This screen displays the developer key and meter number.
+
+Once the developer key and meter number are determined, enter them in the :guilabel:`Developer Key`
+and :guilabel:`Meter Number`fields on the :guilabel:`Fedex Configuration` tab of the shipping
+method.
+
+Shipping details
+~~~~~~~~~~~~~~~~
+
+The main section of the :guilabel:`Fedex Configuration` tab includes a number of additional fields
+used provide information about the shipping method:
+
+- :guilabel:`Fedex Service Type`: the FedEx service used to ship a package.
+- :guilabel:`Fedex Drop-Off Type`: the method for getting a package into FedEx's possession.
+- :guilabel:`Fedex Package Type`: the type of package used for the shipping method.
+- :guilabel:`Package Weight Unit`: the unit of measure used to weigh packages.
+- :guilabel:`Package Length Unit`: the unit of measure used to determine the dimensions of packages.
+- :guilabel:`Label Type`: the type of shipping label used for packages.
+- :guilabel:`Label Format`: the file format used by Odoo to generate shipping labels.
+- :guilabel:`Commercial Invoice Type`: the dimensions and type of the paper used to print invoices.
+
+.. important::
+   The options that should be selected on the :guilabel:`Configuration` tab of a shipping method
+   depend on the negotiated shipping services of the associated FedEx account. To confirm the
+   available services for a FedEx account, visit the :guilabel:`Account Management` page after
+   logging in to the FedEx website, or speak with a customer service representative.
+
+Options
+~~~~~~~
+
+The :guilabel:`Options` section of the :guilabel:`Fedex Configuration` tab provides a few additional
+options to further configure the shipping method:
+
+- :guilabel:`Saturday Delivery`: enable the checkbox to allow packages shipped with the delivery
+  method to be delivered on Saturdays.
+- :guilabel:`Generate Return Label`: enable the checkbox to automatically generate a return label
+  upon validation of a delivery order.
+- :guilabel:`Duties paid by`: use the drop-down menu to select whether duty charges should be paid
+  by the :guilabel:`Sender` or :guilabel:`Recipient`.
+
+Activate shipping method
+========================
+
+By default, shipping methods in Odoo are created within a *test environment*. This means they can
+only be used for testing purposes, and are unable to generate actual shipping orders.
+
+To activate a shipping method in a *production environment*, click the :icon:`fa-stop`
+:guilabel:`Test Environment` smart button at the top of the screen. After doing so, the smart
```

Comment:
Since we're in a new section, I added a bit more details to improve the ease of skimmability.
```suggestion
:guilabel:`Test Environment` smart button at the top of the shipping method form. After doing so, the smart
```